xml/cap_depl_eirini.xml

```diff
@@ -23,6 +23,11 @@
   and <link xlink:href="http://eirini.cf"/>
  </para>
 
+ <para>
+  Different schedulers and stacks have different memory requirements.
+  Not every combination is tested so there is no universal memory setting for &cap;.
```

Comment:
Applies to both files:

Need to mention this is the memory of applications. (Set via `-m` during `cf push` or in the app's manifest https://docs.cloudfoundry.org/devguide/deploy-apps/manifest-attributes.html#memory)

The `Not every combination is tested so there is no universal memory setting for &cap;.` sentence should probably be changed to indicate the memory difference can vary depending on the app deployed and it's up to the user to adjust the setting based on their app.